src/func.d

```diff
@@ -1286,12 +1287,99 @@ public:
             return;
         assert(semanticRun <= PASSsemantic2);
         semanticRun = PASSsemantic2;
+
+        //printf("FuncDeclaration::semantic2 [%s] fd0 = %s %s\n", this.loc.toChars(), this.toChars(), this.type.toChars());
+
+        if (overnext && !errors)
+        {
+            OutBuffer buf1;
+            OutBuffer buf2;
+
+            // Always starts the lookup from 'this', because the conflicts with
+            // previous overloads are already reported.
+            auto f1 = this;
```

Comment:
It's just for the code reading. To me `f1` vs `f2` is easier to understand than `this` vs `f2`.